index.html

```diff
@@ -47,9 +48,14 @@ <h2 id="project_tagline">A pairing and mentoring group for software engineers (R
 <h3>
 <a id="location" class="anchor" href="#location" aria-hidden="true"><span class="octicon octicon-link"></span></a>Location</h3>
 
-<p>CoverMyMeds<br>
-2 Miranova Pl, Floor 10<br>
-Columbus, OH  </p>
```

Comment:
I think it would be useful if we still kept the address in plain text (unless if it's easy to find in the Google Maps widget).